terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf

```diff
@@ -176,9 +177,7 @@ module "server_containerized" {
   }
 
   server_mounted_mirror = "minima-mirror-ci-bv.mgr.suse.de"
-  main_disk_size        = 20
-  repository_disk_size  = 3072
-  database_disk_size    = 150
+  main_disk_size = 3000
```

Comment:
this is incrorrect, for 2 reasons:

* using separate partitions for packages and database is good practice
* the problem in not on main disk (/) but on /var, so it won't work

the problem seems to arise from the initial partitioning of a sle micro

Reply:
Makes sense. Thank you for the comment. We discussed this in a huddle and concluded that I will revert this change for now until we know how to fix it.